src/modules/handleSearchInput.js

```diff
@@ -0,0 +1,17 @@
+import displayItems from './displayItems.js';
+
+const updaterArtistTitle = (artistName) => {
+  const artistTitle = document.body.querySelector('.dra');
+  artistTitle.innerHTML = artistName;
+};
+
+const handleSearchInput = (event) => {
+  if (event.key === 'Enter') {
+    displayItems(event.target.value);
+    updaterArtistTitle(event.target.value);
+  }
+};
+
+document.getElementById('artistInput').addEventListener('keypress', handleSearchInput);
```

Comment:
Kindly move line 15 to the end of `index.js` file. See the comment (2) for that file.